holoviews/plotting/plot.py

```diff
@@ -597,6 +610,9 @@ class GenericElementPlot(DimensionedPlot):
     logy = param.Boolean(default=False, doc="""
         Whether the y-axis of the plot will be a log axis.""")
 
+    padding = param.Number(default=0, doc="""
+        Amount of padding to apply to data ranges.""")
```

Comment:
Some more detail would be good here, at least explaining that it's fractional and whether it is applied only to auto-computed ranges, or also to user-provided ranges.  If the former, maybe "Amount of padding to apply to auto-computed data ranges, as a fraction of the autocomputed range.  E.g. a value of 1.2 will result in a range 20% larger than the auto-computed range."?